lib/model.js

```diff
@@ -106,7 +156,7 @@ var EventEmitter = require('eventemitter3'),
          * @return {Boolean}
          */
         isNew: function () {
-            return !this.id;
+            return !this.getId();
```

Comment:
i think, that it would be better to do like this:
return this._idAttribute.isSet();
to avoid problems with numeric id, that equals to zero, and is converted to false with ! operator